spec/definitions/DataExport.yaml

```diff
@@ -47,13 +47,45 @@ properties:
       criteria:
         $ref: '#/definitions/Condition'
   dateRange:
-    $ref: '#/definitions/DateRange'
+    type: object
+    description: The date range (can be in relative formats).  If omitted, all time will be included.
+    properties:
+      start:
+        type: string
+        description: Any valid datetime arguments including [relative datetime arguments](http://php.net/manual/en/datetime.formats.php).
+        example: 'yesterday'
+      end:
+        type: string
+        description: Any valid datetime arguments including [relative datetime arguments](http://php.net/manual/en/datetime.formats.php).
+        example: 'today'
+    required:
+      - start
+      - end
   emailNotification:
     description: List of emails to be notified when export is completed
     type: array
     items:
       type: string
       format: email
+  fields:
```

Comment:
I wonder if there will be all fields by default (if this field's value is skipped). In this case you should note that in the descriptions so it will be not confusing

Reply:
It will not be all by default.  It will be whatever our default fields are.

Reply:
Let me know if the sentence I added clarifies that at all.